.travis.yml

```diff
@@ -4,10 +4,7 @@ php:
   - 7.1
   - 7.2
   - hhvm
-
-matrix:
-  allow_failures:
-    - php: hhvm
+  - nightly
```

Comment:
`nightly` should probably be an allowed failure!

Reply:
Ok, I add this the ```allow_failures``` back.